diem-unified-charter.md

```diff
@@ -68,12 +62,7 @@ The DIEM WG will work on the following deliverables for the defined scope strict
    The working group will develop a high level architecture and data model but start this work only after the use case and requirements document has reached group consensus. 
    The architecture must not assume any specific serialization formats or securing and discovery mechanisms.
 
-3. A protocol specification describing the binding of emblems to entities (Proposed Standard or Experimental):
-   After the architecture and data model are finished, the working group will specify a single concrete serialization for digital emblems and emblems types for one or two of the initial use cases. 
+3. A protocol specification describing the discovery of digital emblems (Proposed Standard or Experimental):
+   After the architecture and data model are finished, the working group will specify a binding and discovery mechanism addressing one or two of the initial use cases. 
    This specification may describe a mandatory to implement securing mechanism. 
    If a securing mechanism is described, at least one mandatory to implement cryptographic algorithm which is already supported by the securing mechanism must be described as well. 
-   This protocol specification is intended for proposed standard or experimental, and must not assume any specific discovery mechanisms.
-
-4. A specification describing an DNS-based emblem discovery mechanism (Proposed Standard or Experimental):
```

Comment:
I removed this, and the reason is that I think we expect binding and discovery to be related.
I added discovery to the binding PS (I collapsed to PS deliverables into one).